src/transform/split-declaraion.js

```diff
@@ -0,0 +1,28 @@
+import traverser from '../traverser';
+import multiReplaceStatement from '../utils/multi-replace-statement';
+import VariableDeclaration from '../syntax/variable-declaration';
+
+export default function(ast) {
+  traverser.replace(ast, {
+    enter(node, parent) {
+      if (node.type === 'VariableDeclaration' && node.declarations.length > 1) {
+        this.skip();
```

Comment:
that kind of usage is borrowed from [`template` transformer](https://github.com/mohebifar/lebab/blob/master/src/transform/template.js#L11)